src/v0/destinations/mp/transform.js

```diff
@@ -47,21 +46,19 @@ const mPEventPropertiesConfigJson = mappingConfig[ConfigCategory.EVENT_PROPERTIE
 const setImportCredentials = (destConfig) => {
   const endpoint =
     destConfig.dataResidency === 'eu' ? `${BASE_ENDPOINT_EU}/import/` : `${BASE_ENDPOINT}/import/`;
-  const headers = { 'Content-Type': 'application/json' };
   const params = { strict: destConfig.strictMode ? 1 : 0 };
-  const { apiSecret, serviceAccountUserName, serviceAccountSecret, projectId } = destConfig;
-  if (apiSecret) {
-    headers.Authorization = `Basic ${base64Convertor(`${apiSecret}:`)}`;
+  const { serviceAccountUserName, serviceAccountSecret, projectId, token } = destConfig;
```

Comment:
Is apiSecret getting renamed to token here ?

Reply:
This was the condition
```
if (
        !apiSecret &&
        !(serviceAccountUserName && serviceAccountSecret && projectId) &&
        duration.days <= 5
    )
```
Which means -> Configuration should only contain `token` and days should be <= 5